tests/network-setups/fabric/dev/scripts/setupAppCC.sh

```diff
@@ -1,8 +1,7 @@
 CC_NAME=${1:-"simplestate"}
 
-INTEROPCC_CHAINCODE_PATH=$PWD/../../../../core/network
 CHAINCODE_PATH=$PWD/../shared/chaincode
-APP_CC_PATH=$PWD/../../../../samples/fabric/${CC_NAME}
+APP_CC_PATH=$PWD/../../../../samples/fabric/${CC_NAME}$2
```

Comment:
What is this `$2` for?

Reply:
This was an option I'd added earlier and passed in the Makefile, but then thought better of it and removed it. But I forgot to remove the `$2` from the scripts. Will do that.